### DISCUSSION TITLE
Changing the color of operator symbols

### DISCUSSION THREAD

**Discussion by zilverjk** [NvChad/NvChad#3062] [Q&A]
How or what is the correct property to override the color of the operator symbols such as || ! - + => <= 
![image](https://github.com/user-attachments/assets/38a79335-fb4d-4c2e-b516-aea00a88eec3)

![image](https://github.com/user-attachments/assets/7a670f34-2313-46d8-b5d5-20431fa69903)

**Comment by siduck** [comment]
run Inspect command on it and then use hl_override if the hlgroup exists within base46 else hl_add
